sound/soc/sof/intel/hda-dai.c

```diff
@@ -234,7 +234,7 @@ static int hda_dai_link_trigger(struct snd_pcm_substream *substream, int cmd)
 	if (!link)
 		return -EINVAL;
 
-	dev_dbg(cpu_dai->dev, "%s: cmd=%d\n", __func__, cmd);
+	dev_dbg(cpu_dai->dev, "%s: cmd=%\n", __func__, cmd);
```

Comment:
This specific change looks extremely dubious.

Reply:
it does indeed, thanks for pointing this out  @paulstelian97

Reply:
fixed locally.